WaterLog_Backend/Controllers/PumpsController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Storage;
+using Microsoft.Extensions.Configuration;
+using WaterLog_Backend.Models;
+
+namespace WaterLog_Backend.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class PumpsController : ControllerBase
+    {
+        private readonly DatabaseContext _db;
+        readonly IConfiguration _config;
```

Comment:
I've been seeing this constructor everywhere like this. Might be worth looking into moving these two veriables to the base class if every controller is going to have them